nf_core/modules/lint/main_nf.py

```diff
@@ -241,7 +241,9 @@ def check_process_section(self, lines):
         if l.startswith("https://containers") or l.startswith("https://depot"):
             # e.g. "https://containers.biocontainers.pro/s3/SingImgsRepo/biocontainers/v1.2.0_cv1/biocontainers_v1.2.0_cv1.img' :" -> v1.2.0_cv1
             # e.g. "https://depot.galaxyproject.org/singularity/fastqc:0.11.9--0' :" -> 0.11.9--0
-            singularity_tag = re.search("(?:\/)?(?:biocontainers_)?(?::)?([A-Za-z\d\-_\.]+)(?:\.img)?['\"]", l).group(1)
+            singularity_tag = re.search("(?:\/)?(?:biocontainers_)?(?::)?([A-Za-z\d\-_\.]+?)(?:\.img)?['\"]", l).group(
+                1
+            )
```

Comment:
I wonder if we can remove the strange multiline, I guess they are from Prettier, or is the linting failing then?

Reply:
It's 121 characters long, which exceeds the 120 limit. Once the string-stuff is merged, it will be on one line again because two backslashes will be removed and only one `r` added ;-)